lib/containers/helm.pm

```diff
@@ -72,14 +72,24 @@ sub helm_configure_values {
     }
 
     if (my $image = get_var('CONTAINER_IMAGE_TO_TEST')) {
-        my ($repository, $tag) = split(':', $image, 2);
+        my ($registry, $repository, $tag);
         my $helm_values_image_path = get_required_var('HELM_VALUES_IMAGE_PATH');
 
-        # Add space before appending if $set_options already has content
+        if ($image =~ m{^(registry\.suse\.(?:com|de))/([^:]+)(?::(.+))?$}) {
+            # Explicit SUSE registry prefix
+            $registry = $1;
+            $repository = $2;
+            $tag = $3 // 'latest';
```

Comment:
Maybe I misunderstood the regex in line 78, but AFAICS here should not be a default value, because if this isn't present, we would fall into the `} else {` branch, no? :thinking: